src/action.ts

```diff
@@ -215,9 +216,9 @@ class Action {
     }
   }
 
-  private displayFailedTasks(executionGraph: ExecutionGraph, tasks: Task[]): Task[] {
-    const failed: Task[] = []
-    for (const task of tasks.filter(t => t.status === TaskStatus.Failed)) {
+  private displayUnconcludedTasks(executionGraph: ExecutionGraph, tasks: Task[]): Task[] {
+    const unconcluded: Task[] = []
+    for (const task of tasks.filter(t => t.status === TaskStatus.Failed && TaskStatus.Skipped)) {
```

Comment:
Isn't this an OR instead? Could you add a test to check this?

```suggestion
    for (const task of tasks.filter(t => t.status === TaskStatus.Failed || t.status === TaskStatus.Skipped)) {
```